src/extension/wps/src/test/java/au/org/emii/wps/gogoduck/parameter/SubsetParametersTest.java

```diff
@@ -24,6 +24,14 @@ public void testParameterParsing() {
     }
 
     @Test
+    public void testValidTime() {
```

Comment:
Perhaps add a test for no decimal places as well which is what MARVL were providing.